RELEASE.md

```diff
@@ -1,6 +1,57 @@
 # Release 1.6.0
 
-The 1.6 minor series tracks TensorFlow 1.6. Release notes upcoming.
+NOTICE: TensorBoard 1.6.0+ has moved to the `tensorboard` package name on PyPI:
+https://pypi.python.org/pypi/tensorboard. Only bugfix updates on 1.5.x will be
+applied to the old package name (`tensorflow-tensorboard`). To upgrade to
+TensorBoard 1.6.0+ we suggest you *first* `pip uninstall tensorflow-tensorboard`
+before doing `pip install tensorboard`. See "Known Issues" below if you run into
+problems using TensorBoard after upgrading.
+
+The 1.6 minor series tracks TensorFlow 1.6.
+
+## Highlights
+
+- (beta release) New Profile dashboard, which provides a suite of tools for
+  inspecting TPU performance.  See for details:
+  https://github.com/tensorflow/tensorboard/tree/1.6/tensorboard/plugins/profile
```

Comment:
Optional Hint: I like to sometimes put `<http://blah>` notation around markdown links, which is more proper and compatible IIRC.

Reply:
Ack.  I'd prefer not to, since it's unnecessary in most modern markdown renderers and in some it's definitely less compatible in that it might get interpreted as an unknown HTML tag and stripped rather than just rendered in plain text, which is a worse failure mode.

https://meta.stackexchange.com/questions/266263/links-in-angle-brackets-dont-show-up
https://github.com/jgm/peg-markdown/issues/33